docs/faq.md

```diff
@@ -14,6 +14,8 @@ If a question you have is not answered below, please [submit an issue](/../../is
 [Can I ADD a custom directory to the image?](#can-i-add-a-custom-directory-to-the-image)\
 [Can I build to a local Docker daemon?](#can-i-build-to-a-local-docker-daemon)\
 [I am seeing `ImagePullBackoff` on my pods.](#i-am-seeing-imagepullbackoff-on-my-pods-in-minikube)\
+[How do I configure a proxy?](#how-do-i-configure-a-proxy)\
+[How can I examine network traffic?](#how-can-i-examine-network-traffic)\
```

Comment:
Why no entry for "How can I diagnose problems pulling or pushing from remote registries?"